### PR TITLE
chore(fr): Updates small changes up to EN dbf366a

### PR DESCRIPTION
**Pull request by pacyL2K19**
## PR Checklist
Merci de vérifier que vous suivez bien les recommandations suivantes :

- [x] Les messages de commits suivent bien les recommandations : https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md
- [x] L'homogénéité de la traduction est respectée (guide à venir)
- [x] Les erreurs communes sont évitées (redirection vers la version EN, traduction des ancres, ...)

## Types de PR
Quel est l'objet de cette PR :

- [ ] Traduction d'une nouvelle page
- [ ] Modification d'une page déjà traduite
- [x] màj pour suivre version EN dbf366a

## Liens 

#163

### CODE REVIEW

**Comment by pacyL2K19** [comment]
@Lou8is 
Kindly have a look

File: content/fundamentals/execution-context.md [hunk 1]
```diff
@@ -262,23 +242,100 @@ export class CatsController {}
...
+@@switch
+@Post()
+@SetMetadata('roles', ['admin'])
+@Bind(Body())
+async create(createCatDto) {
+  this.catsService.create(createCatDto);
+}
+```
+
+> info Astuce Le décorateur @SetMetadata() est importé depuis le package @nestjs/common.
```
Comment (Lou8is) [inline content/fundamentals/execution-context.md:280]: laisser les "**" autour de Astuce (=> **Astuce**)

File: content/fundamentals/execution-context.md [hunk 2]
```diff
@@ -262,23 +242,100 @@ export class CatsController {}
...
+
+Avec la construction ci-dessus, nous avons attaché les métadonnées roles (roles est une clé de métadonnées et ['admin'] est la valeur associée) à la méthode create(). Bien que cela fonctionne, il n'est pas recommandé d'utiliser @SetMetadata() directement dans vos routes. Au lieu de cela, vous pouvez créer vos propres décorateurs, comme illustré ci-dessous :
```
Comment (Lou8is) [inline content/fundamentals/execution-context.md:282]: Laisser les `` aux mots-clés (=> `roles`, => `create()`, etc)

File: content/fundamentals/execution-context.md [hunk 3]
```diff
@@ -262,23 +242,100 @@ export class CatsController {}
...
+import { SetMetadata } from '@nestjs/common';
+
+export const Roles = (...roles: string[]) => SetMetadata('roles', roles);
+@@switch
+import { SetMetadata } from '@nestjs/common';
+
+export const Roles = (...roles) => SetMetadata('roles', roles);
+```
+
+Cette approche est beaucoup plus propre et lisible, et ressemble en quelque sorte à l'approche Reflector#createDecorator. La différence réside dans le fait qu'avec @SetMetadata, vous avez plus de contrôle sur la clé et la valeur des métadonnées, et vous pouvez également créer des décorateurs qui prennent plus d'un argument.
```
Comment (Lou8is) [inline content/fundamentals/execution-context.md:295]: idem pour les ``

File: content/fundamentals/execution-context.md [hunk 4]
```diff
@@ -262,23 +242,100 @@ export class CatsController {}
...
+@@switch
+@Post()
+@Roles('admin')
+@Bind(Body())
+async create(createCatDto) {
+  this.catsService.create(createCatDto);
+}
+```
+
+Pour accéder aux rôles de la route (métadonnées personnalisées), nous utiliserons à nouveau la classe d'aide Reflector :
```
Comment (Lou8is) [inline content/fundamentals/execution-context.md:315]: idem pour les ``

File: content/fundamentals/execution-context.md [hunk 5]
```diff
@@ -262,23 +242,100 @@ export class CatsController {}
...
+@Injectable()
+@Dependencies(Reflector)
+export class CatsService {
+  constructor(reflector) {
+    this.reflector = reflector;
+  }
+}
+```
+
+> info Astuce La classe Reflector est importée depuis le package @nestjs/core.
```
Comment (Lou8is) [inline content/fundamentals/execution-context.md:333]: ** autour de Astuce

File: content/fundamentals/execution-context.md [hunk 6]
```diff
@@ -262,23 +242,100 @@ export class CatsController {}
...
+
+Maintenant, pour lire les métadonnées du gestionnaire, utilisez la méthode get().
+
+```typescript
+const roles = this.reflector.get<string[]>('roles', context.getHandler());
+```
+
+Ici, au lieu de passer une référence au décorateur, nous passons la clé des métadonnées en tant que premier argument (dans notre cas, c'est 'roles'). Tout le reste reste identique à l'exemple `Reflector#createDecorator`.
```
Comment (Lou8is) [inline content/fundamentals/execution-context.md:341]: ** de "clé" et `` de roles

**Review by Lou8is — CHANGES_REQUESTED** [review]
Hi, translation looks good to me!

I noted a few errors to correct before approval (mostly formatting using ** or `` ).

Thanks for your time!

**Comment by Lou8is** [comment]
The build step seems to be broken due to an error on the updated file, I suggest you test it locally to correct the error.

If you're stuck I'll be happy to help :)

**Comment by pacyL2K19** [comment]
> Hi, translation looks good to me!
> 
> I noted a few errors to correct before approval (mostly formatting using ** or `` ).
> 
> Thanks for your time!

Thank you for the review
I am on it

**Comment by pacyL2K19** [comment]
> > Hi, translation looks good to me!
> > I noted a few errors to correct before approval (mostly formatting using ** or `` ).
> > Thanks for your time!
> 
> Thank you for the review I am on it

@Lou8is 
I have fixed the formatting issues, thank you for catching them
About the build step, I only updated the md file and locally the build runs successfully 
Can you help me check this?
Screenshot 2023-11-20 at 21 47 50

**Review by Lou8is — APPROVED** [review]

**Comment by Lou8is** [comment]
The updated file was building OK, it might have been the missing ** for Hint macros that prevented it before. Your updates should be live in a few minute, thanks!

**Pull request merged**